htlcswitch/link.go

```diff
@@ -1275,13 +1275,13 @@ func (l *channelLink) randomFeeUpdateTimeout() time.Duration {
 	return time.Duration(prand.Int63n(upper-lower) + lower)
 }
 
-// handleDownStreamPkt processes an HTLC packet sent from the downstream HTLC
+// handleDownstreamPkt processes an HTLC packet sent from the downstream HTLC
 // Switch. Possible messages sent by the switch include requests to forward new
 // HTLCs, timeout previously cleared HTLCs, and finally to settle currently
 // cleared HTLCs with the upstream peer.
 //
 // TODO(roasbeef): add sync ntfn to ensure switch always has consistent view?
-func (l *channelLink) handleDownStreamPkt(pkt *htlcPacket, isReProcess bool) {
+func (l *channelLink) handleDownstreamPkt(pkt *htlcPacket) {
```

Comment:
Is this an historical left-over of was accidentally some test coverage deleted in a previous commit?

Reply:
i think it was just left over from some prior refactor. the name change is just done to match `handleUpstreamPkt` because the extra `S` bothers me lol